www/includes/easyparliament/templates/html/mp/divisions.php

```diff
@@ -15,14 +15,19 @@
         </div>
         <div class="person-panels">
             <div class="sidebar__unit in-page-nav">
-                <?php if ( isset($policydivisions) && $policydivisions && count($policydivisions) == 1 ) { ?>
-                <p class="policy-votes-intro">
-                    How <?= $full_name ?> voted on <?= $policydivisions[array_keys($policydivisions)[0]]['desc'] ?>.
-                </p>
-                <?php } ?>
-                <ul>
-                    <li><a href="<?= $member_url ?>/votes">Back to all topics</a></li>
-                </ul>
+                <div>
+                    <?php if ( isset($policydivisions) && $policydivisions && count($policydivisions) == 1 ) { ?>
+                        <p class="policy-votes-intro">
+                            How <?= $full_name ?> voted on <?= $policydivisions[array_keys($policydivisions)[0]]['desc'] ?>.
+                        </p>
+                        <?php } ?>
```

Comment:
If you're going to change indents (not sure it matters much in this type of file), this one and below is wrong :)